### PR TITLE
Fix broken link for Windows Phone Course and move it to the courses s…

### PR DESCRIPTION
**Pull request by loudercake**
…ection

## What does this PR do?
Improve repo

### Description
Move a course to the course's section, where it was previously on the book section, plus using the archived version since the original link is now dead.
### Why is this valuable (or not)?
It makes the repo more organized.
### How do we know it's really free?
Since it used to be accessible on Microsoft Channel 9 website.
### For book lists, is it a book? For course lists, is it a course? etc.
Course.
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
the archive is not complete enough, and it's not useful except for historians. we should just delete the link

**Comment by loudercake** [comment]
What exactly about the archive isn't complete? I gave it a glance and it seems complete to me. Besides, the usefulness is relative, as people could use the course as reference to other things related to .NET.

**Comment by loudercake** [comment]
All requests have been done.

**Comment by eshellman** [comment]
I clicked on "Part 10" and the video on that page appears not to have been archived.

**Comment by loudercake** [comment]
[Part 10 has been archived and can be easily accessed by just clicking.](https://web.archive.org/web/20150219201517/http://channel9.msdn.com/Series/Windows-Phone-8-1-Development-for-Absolute-Beginners/Part-10-Exercise-Tip-Calculator-as-a-Universal-App)

**Comment by loudercake** [comment]
do i require any more maintenance?

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -1294,6 +1295,12 @@
 * [Web Programming](https://open.cs.uwaterloo.ca/web-programming/) - Centre for Education in Math and Computing (University of Waterloo)
 
 
+### Windows Phone
+
+* [Windows Phone 8.1 Development for Absolute Beginners](https://web.archive.org/web/20150213035325/http://channel9.msdn.com/Series/Windows-Phone-8-1-Development-for-Absolute-Beginners) - Bob Tabor, Matthias Shapiro, Larry Lieberman (Channel 9, MSDN) *(:card_file_box: archived)*
```
Comment (eshellman) [inline courses/free-courses-en.md:1300]:
```suggestion
* [Windows Phone 8.1 Development for Absolute Beginners](https://web.archive.org/web/20150213035325/http://channel9.msdn.com/Series/Windows-Phone-8-1-Development-for-Absolute-Beginners) - Bob Tabor, Matthias Shapiro, Larry Lieberman *(:card_file_box: archived)*
```

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**